blog/_posts/2017-09-13-Blog-1team-Serv0101.md

```diff
@@ -0,0 +1,38 @@
+---
+title: “Blog- Team Serv0101”
+layout: post
+created_at: Wed Sept 13 2017
+permalink: blog/2017-09-13-final-blog-post-team-Serv0101
+current: blog
+author: Rakhi Sharma And Neha Yadav
+twitter: @atbrakhi
+categories:
+- student-posts
+---
+
+## Blog- Team Serv0101
```

Comment:
There is no text between the title and this heading. So, it looks bit odd to. Change either of them or remove the heading.